deepface/DeepFace.py

```diff
@@ -462,7 +462,7 @@ def stream(
             OpenFace, DeepFace, DeepID, Dlib, ArcFace, SFace and GhostFaceNet (default is VGG-Face).
 
```

Comment:
you added some facial recognition models for yolo but they are missing here

Reply:
fixed

Reply:
but facial recognition model names in modeling are not Yolo-Face